src/mol-gl/compute/grid3d.ts

```diff
@@ -191,23 +194,31 @@ export function createGrid3dComputeRenderable<S extends RenderableSchema, P, CS>
 
                 if (spec.cumulative.yieldPeriod && i !== states.length - 1) {
                     if (i % yieldPeriod === yieldPeriod - 1) {
-                        webgl.readPixels(0, 0, 1, 1, array);
+                        webgl.waitForGpuCommandsCompleteSync();
```

Comment:
Wasn't the reason for the readPixel that the wait wasn't working propertly?

Reply:
`webgl.waitForGpuCommandsComplete` doesn't work well, too slow here

`webgl.waitForGpuCommandsCompleteSync` is just doing the readPixels thing, so the change is mostly cosmetic